openedx/core/djangoapps/credit/views.py

```diff
@@ -29,6 +29,59 @@
 log = logging.getLogger(__name__)
 
 
+@require_GET
+def get_provider_detail(request, provider_id):
+    """
+    get the details of required provider
+
+    This end-point will get the details of providers against
```

Comment:
This additional sentence can be eliminated as it doesn't relay much more information than the one above.